moveit_core/profiler/src/profiler.cpp

```diff
@@ -174,7 +174,7 @@ void Profiler::console()
   std::stringstream ss;
   ss << std::endl;
   status(ss, true);
-  RCUTILS_LOG_INFO("profiler", ss.str().c_str());
+  RCUTILS_LOG_INFO_NAMED("profiler", "%s", ss.str().c_str());
```

Comment:
Weird that this was changed to use `RCUTILS` instead of `rclcpp::Logger`. We will probably replace it when refactoring logging.

Reply:
I changed this to `rclcpp::Logger`.

Reply:
Great